### ISSUE TITLE
matlab mex problem

### ISSUE THREAD

**Issue by mekides** [kbitr/matlab-yolov2#1]
I am getting:
MEX cannot find library 'yolo' specified with the -l option.
 MEX looks for a file with one of the names:
 libyolo.lib
 yolo.lib

What seems to be the problem? I run the makefile using cygwin, and I could not find the mentioned files.

**Comment by kbitr** [comment]
Hey, I don't think anybody tried this on Windows, yet. I will look into it next week.

Saying you can make the file using cygwin without errors?

**Comment by mekides** [comment]
No. I could not make them. That is the problem. I think it would be great if there is alternative method for windows. Many people would appreciate it.

With many thanks,

**Comment by kbitr** [comment]
Well, yes. This is based on the original pjreddie/darknet code. There is a Windows port AlexeyAB/darknet - you can try to change the paths in the makefile and c-files and then make it.

That's what I wanted to try this week.

**Comment by mekides** [comment]
OK, I will try that. 

Thank you!

**Comment by kbitr** [comment]
Did not find a solution this week, will continue next week.

Actually, it might be worth a consideration to just use [AlexeyAB/yolo2_light](https://github.com/AlexeyAB/yolo2_light) or the very new direct [opencv/opencv](https://github.com/opencv/opencv) implementation. Will see!

**Comment by kbitr** [comment]
Hey @mekides, did you put any more work into this? I am and I will probably release a Windows version the next days (whenever it works, no idea).

**Comment by mekides** [comment]
Hey, not really. I had to move to other urgent work on python. I will come back to it when I am done. Thanks for all your help.

**Comment by kbitr** [comment]
- Windows port put on ice for now -
